app/src/main/java/org/wikipedia/settings/dev/DeveloperSettingsPreferenceLoader.kt

```diff
@@ -214,6 +216,17 @@ internal class DeveloperSettingsPreferenceLoader(fragment: PreferenceFragmentCom
             activity.startActivity(Intent(activity, CategoryDeveloperPlayGround::class.java))
             true
         }
+        findPreference(R.string.preference_key_recommended_reading_list_notification_enabled)
+            .setOnPreferenceChangeListener { _, newValue ->
+                val selectedFrequency = newValue as String
+                val source = when (selectedFrequency) {
```

Comment:
We already have an enum called `RecommendedReadingListUpdateFrequency` that defines these frequencies. Can we use the enum here (its values _and names_!) instead of hardcoding more strings and arrays?